dist/index.html

```diff
@@ -14,40 +14,79 @@
     <title>TO-DO</title>
```

Comment:
Validate your html with [https://validator.w3.org/#validate_by_input](https://validator.w3.org/#validate_by_input), and work on its warning and suggestions